cmd/kubelet/app/options/options.go

```diff
@@ -94,6 +95,11 @@ func NewKubeletServer() *KubeletServer {
 	api.Scheme.Default(versioned)
 	config := componentconfig.KubeletConfiguration{}
 	api.Scheme.Convert(versioned, &config, nil)
+
+	config.CAdvisorStatsCacheDuration = metav1.Duration{Duration: 2 * time.Minute}
```

Comment:
Does not make sense to do initialization here.
Prefer to keep in `cadvisor_linux.go`. If these values are not set, use default constant values instead.